releaseNotes/AzureSpringCloudAppConfigurationConfig.md

```diff
@@ -0,0 +1,38 @@
+# azure-spring-cloud-appconfiguration-config
+
+[Source code][source_code] | [Package (Maven)][package] | [README][readme] | [Product documentation][docs] | [Samples][samples]
+
+# azure-spring-cloud-appconfiguration-config-web
+
+[Source code web][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
+
+## 2.0.0-beta.1 - May 5, 2021
+
+* Enables loading of configurations from multiple config stores.
+* Each store can load configurations by using key/label `selects`. Many `selects` can be used, default is key = `/applicaiton/*` with label = `${spring.profiles.active}` or if null `\0`. See [README][readme] for full configuration.
+* Authentication vis Connection String, Managed Identity, or any method supported by Azure Identity SDK, see Token Credential Provider in README.
```

Comment:
"vis" - typo?

Don't we support this already or are you trying to have a full list of all features?